src/index.html

```diff
@@ -2,11 +2,267 @@
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="card__rate">
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <button class="card__button-buy" data-qa="card-hover">buy</button>
+        </div>
+
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__image"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="card__rate">
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <button class="card__button-buy" data-qa="card-hover">buy</button>
+        </div>
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__image"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="card__rate">
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <button class="card__button-buy" data-qa="card-hover">buy</button>
+        </div>
+        <div class="card">
```

Comment:
Add empty line between the all cards
```suggestion
        </div>
        
        <div class="card">
```